steering_committee.md

```diff
@@ -146,3 +146,8 @@ The social media team is responsible for the outreach of the community, by tweet
 ## Engagement team
 
 The engagement officers are responsible for coming up with new initiatives (projects, summer schools, outreach activities, etc.) and propose them to the community. They will also oversee ongoing initiatives and provide support for them to reach their goals. They will ensure that the initiative is known in the community, as well as outside the community together with the media team. As well as suggesting potential funding sources for the initiative, by engaging with the treasurer team.
+
+## IDEA (Inclusion, Diversity, Equity, and Accessibility) Coordinator (NEW!)
```

Comment:
```suggestion
## IDEA (Inclusion, Diversity, Equity, and Accessibility) Coordinator
```
I like the idea of highlighting this, but we do not update the website regularly enough, so that will likely be `(NEW!) for a while if we leave it there. I think the News post should be enough of an update, since it will also show on the landing page.